scripts/docker-entrypoint.sh

```diff
@@ -29,6 +29,13 @@ mkdir -p $config_root
 secret_root="${pega_root}/secrets"
 mkdir -p $secret_root
 
+tls_cert_root="${pega_root}/tlscerts"
+mkdir -p $tls_cert_root
+
+tomcat_cert_root="${pega_root}/tomcatcerts"
+mkdir -p $tomcat_cert_root
+
```

Comment:
Is two directories required? tls_cert_root & tomcat_cert_root? Can the files be directly mounted to `tomcat_cert_root`?

Reply:
I wanted to decode a file inside that, but i was not able to since it is a link, so created another directory for that.
check line 77